debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumKafkaContainer.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.testcontainers;
+
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.DockerImageName;
+
+public class DebeziumKafkaContainer {
+    private static final String defaultImage = "confluentinc/cp-kafka:7.2.10";
```

Comment:
I don't think we should use Confluent's Kafka container / Confluent Platfrom? @jpechane

Reply:
@rk3rn3r We have been using it for the whole time. It was just hidden in Kafka Container default logic. Let me check if there is a possibility to easily swap to apache one.

Reply:
Sure, I think so. The main reason is that confluent provides a very good configuration that properly handles docker/cloud scenarios with external and internal IPs/hostnames etc which, at least in the past, was missing from the original Apache Kafka container. I don't know if they closed this gap.

Reply:
TBH, if it would rquire a major effort then I am fine with keeping things as they are. For me it is more important for testing that we use our Connect container.

Reply:
So, I have checked the Apache container for TC. It seems that we would have to upgrade TC dependency and it is still a little bit behind. For example, it does not support KRaft, which TBH significantly speeds up the deployment for us. I think we should keep this as is so we can migrate the rest of the TF images to CentOS Stream 9 and create Jira to revisit this later with proper investigation. WDYT @jpechane @rk3rn3r

Reply:
Honestly, I would very much appreciate merging this, as it is the only thing that is blocking me from rebuilding the rest of the TF slaves, and we can be calm about the deprecation of repositories there...